packages/pigeon/platform_tests/flutter_null_safe_unit_tests/lib/null_safe_pigeon.dart

```diff
@@ -202,7 +199,32 @@ class Api {
         details: error['details'],
       );
     } else {
-      return SearchRequests.decode(replyMap['result']!);
+      return (replyMap['result'] as SearchRequests?)!;
+    }
+  }
+
+  Future<int> inc(int arg) async {
```

Comment:
It did actually get renamed https://github.com/flutter/packages/pull/414/files#diff-9e91b13fda610a88abcd78209d839aa424dfcda5975f24bb91449c60d8ce68a2R15

This is generated code I didn't check in (that we want to remove at some point but can't because of the analysis step).  I'll update the code.

Reply:
Done, turns out there were actually 2 instances of 'inc'